Instastories.py

```diff
@@ -12,29 +12,37 @@
 except ImportError as e:
     PRINT_TABLE = False
 
-#  #  #  #  #
+#####
 
 COOKIE =      {  "cookie": None,
                  "user-agent": "Instagram 10.3.2 (iPhone7,2; iPhone OS 9_3_3; en_US; en-US; scale=2.00; 750x1334) AppleWebKit/420+",
                  "cache-control": "no-cache" }
 
+EXTRA_ID = [None] # Get stories from unfollowed users by using their ID
+#EXTRA_USR = ["xxxx", "yyyy", "zzzz"] # Get stories from unfollowed users by using their Nicknames, deprecated due to Instagram changes
 
 
-#EXTRA_ID = [X X X X X X X X, X X X X X X X X, X X X X X X X X] # Get stories from unfollowed users by using their ID
-#EXTRA_USR = ["x x x x", "y y y y", "z z z z"] # Get stories from unfollowed users by using their Nicknames, deprecated due to Instagram changes
+# Get the optional token path argument from the command line
 
 def getCookie():
-	cookie_path = sys.argv[2]
-	cookie_file = open(cookie_path, "r")
-	cookie = cookie_file.read()
-	COOKIE["cookie"] = cookie
-	cookie_file.close()
+    token =      {
+             "cookie": None,
+             "user-agent": "Instagram 10.3.2 (iPhone7,2; iPhone OS 9_3_3; en_US; en-US; scale=2.00; 750x1334) AppleWebKit/420+",
+             "cache-control": "no-cache" }
 
-	while 1:
-		if COOKIE.get("cookie") == None:
-			print("Invalid token, try again")
-		else:
-			break
+    if __name__ == "__main__":
+        parser = argparse.ArgumentParser(description="TOKEN PATH AND DESTINATION FOLDER")
+        parser.add_argument("t", metavar="<token>", help="Insert the path of cookie file")
+        args = parser.parse_args()
+        cookie_path = args.t 
+        with open(cookie_path, "r") as f:
+            token["cookie"] = f.read()
+        return token
+
+    else: 
+        with open("token.txt", "r") as f:
+            token["cookie"] = f.read()
+            return token
```

Comment:
The default logic (used when you launch it from the server script) should be in the server side, not here.